rosbot_description/urdf/components/vl53lox.urdf.xacro

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="utf-8"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro">
+  <!-- vl53lox defining macro -->
+  <xacro:macro name="vl53lox" params="parent xyz rpy prefix use_gpu">
```

Comment:
`use_gpu` probably should be also removed (and other unnecessary references to it).